src/components/header/SearchBar.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { GiMagnifyingGlass } from 'react-icons/gi'
+import styles from './header.module.scss'
+
+function SearchBar(props) {
+  return (
+    <div className={styles.searchBar}>
+        <GiMagnifyingGlass />
+        <input type='text' placeholder='wizeline' style={{border: 'none', background: 'none'}}/>
```

Comment:
Intentar agregar lo menos posible style-in-line, si ya obtienes estilos de una importación, mejor todo traerlo de ahí